### PR TITLE
Feat: Align go-redis ProbabilisticCmdable

### PR DESCRIPTION
**Pull request by unknowntpo**
This PR introduced generic `rueidiscompat.baseCmd` to avoid writing boilerplate code like `.Val()`, `.Err()`, `Result()` for commands.

And used `rueidiscompat.Scan` to scan result into `go-redis` structure.

### CODE REVIEW

File: rueidiscompat/adapter.go [hunk 1]
```diff
@@ -2970,6 +3046,473 @@ func (c *Compat) TFCallASYNCArgs(ctx context.Context, libName string, funcName s
...
+	return newStatusCmd(c.client.Do(ctx, cmd))
+}
+
+func (c *Compat) CFScanDump(ctx context.Context, key string, iterator int64) *ScanDumpCmd {
+	cmd := c.client.B().CfScandump().Key(key).Iterator(iterator).Build()
+	return newScanDumpCmd(c.client.Do(ctx, cmd))
+}
+
+func (c *Compat) CFLoadChunk(ctx context.Context, key string, iterator int64, data interface{}) *StatusCmd {
+	cmd := c.client.B().CfLoadchunk().Key(key).Iterator(iterator).Data(fmt.Sprint(data)).Build()
```
Comment (rueian) [inline rueidiscompat/adapter.go:3303]:
```suggestion
	cmd := c.client.B().CfLoadchunk().Key(key).Iterator(iterator).Data(str(data)).Build()
```

File: rueidiscompat/adapter.go [hunk 2]
```diff
@@ -2970,6 +3046,473 @@ func (c *Compat) TFCallASYNCArgs(ctx context.Context, libName string, funcName s
...
+	return newStatusCmd(c.client.Do(ctx, cmd.Build()))
+}
+
+func (c *Compat) BFScanDump(ctx context.Context, key string, iterator int64) *ScanDumpCmd {
+	cmd := c.client.B().BfScandump().Key(key).Iterator(iterator).Build()
+	return newScanDumpCmd(c.client.Do(ctx, cmd))
+}
+
+func (c *Compat) BFLoadChunk(ctx context.Context, key string, iterator int64, data interface{}) *StatusCmd {
+	cmd := c.client.B().BfLoadchunk().Key(key).Iterator(iterator).Data(fmt.Sprint(data)).Build()
```
Comment (rueian) [inline rueidiscompat/adapter.go:3179]:
```suggestion
	cmd := c.client.B().BfLoadchunk().Key(key).Iterator(iterator).Data(str(data)).Build()
```

File: rueidiscompat/adapter.go [hunk 3]
```diff
@@ -2970,6 +3046,473 @@ func (c *Compat) TFCallASYNCArgs(ctx context.Context, libName string, funcName s
...
+
+func (c *Compat) CFLoadChunk(ctx context.Context, key string, iterator int64, data interface{}) *StatusCmd {
+	cmd := c.client.B().CfLoadchunk().Key(key).Iterator(iterator).Data(fmt.Sprint(data)).Build()
+	return newStatusCmd(c.client.Do(ctx, cmd))
+}
+
+func (c *Compat) CMSIncrBy(ctx context.Context, key string, elements ...interface{}) *IntSliceCmd {
+	_cmd := c.client.B().CmsIncrby().Key(key)
+	for i := 0; i < len(elements); i += 2 {
+		_cmd.Item(fmt.Sprint(elements[i])).Increment((int64)(elements[i+1].(int)))
```
Comment (rueian) [inline rueidiscompat/adapter.go:3310]:
```suggestion
		_cmd.Item(str(elements[i])).Increment((int64)(elements[i+1].(int)))
```

**Comment by rueian** [comment]
Thank you for this great work @unknowntpo!

**Comment by unknowntpo** [comment]
@rueian Thanks for reviewing.

**Pull request merged**